src/components/home/homeMidSection/homeMidSection.tsx

```diff
@@ -1,114 +1,229 @@
+                className="d-inline-block shadow border-muted mx-2 p-1"
+              >
+                <Card.Body>
+                  <Card.Title className="text-secondary fw-bold">
+                    <Row
+                      style={{ fontSize: "36px" }}
+                      className="p-0 col-auto mb-2"
+                    >
+                      <span className="fa-layers fa-fw fa-lg">
+                        <FontAwesomeIcon icon={faCircle} />
+                        <FontAwesomeIcon icon={faDna} transform="shrink-8" />
+                      </span>
+                    </Row>
+                    <Row className="w-bold fs-5 ps-0">
+                      <span className="text-center">
+                        Platforms:&nbsp;
+                        {
+                          Object.keys(summary.protocol_summary.stats.protocol)
+                            .length
+                        }
+                      </span>
+                    </Row>
+                  </Card.Title>
+                  <Card.Text as="div">
+                    <Row className="pt-3 fs-7 ">
+                      <Col>
+                        {getItemsForSummary(
+                          summary.protocol_summary.stats.protocol
+                        ).map((x) => {
+                          return (
+                            <Row
+                              key={x}
+                              className="text-capitalize ms-0 ps-0 mb-2"
+                            >
+                              <Col className="ms-0 ps-0">
+                                {x.split(": ")[0]}:
+                              </Col>
+                              <Col className="col-auto text-end fw-bold">
+                                {x.split(": ")[1]}
+                              </Col>
+                            </Row>
+                          );
+                        })}
+                      </Col>
+                    </Row>
+                  </Card.Text>
+                </Card.Body>
+              </Card>
+              <Card
+                style={{ width: "16rem", borderRadius: "15px" }}
+                className="d-inline-block shadow border-white mx-2 bg-primary p-1"
+              >
+                <Card.Body className="text-white">
+                  <Card.Title>
+                    <Row
+                      style={{ fontSize: "36px" }}
+                      className="p-0 col-auto mb-2"
+                    >
+                      <span className="fa-layers fa-fw fa-lg">
+                        <FontAwesomeIcon icon={faCircle} />
+                        <FontAwesomeIcon icon={faUser} transform="shrink-8" />
+                      </span>
+                    </Row>
+                    <Row className="w-bold fs-5 ps-0">
+                      <span className="text-center">
+                        Individuals:&nbsp;
+                        {summary.individual_summary.count}
```

Comment:
Could you add individual icons as before? The ELSI team said that icons are ok.

Reply:
In another PR.